src/frontend/src/styles/main.css

```diff
@@ -1131,6 +1131,9 @@ by all browsers (FF is missing) */
 }
 
 .c-toast-body__message {
+  /* Ensures the content doesn't grow out of the toast or push other items */
+  overflow-x: auto;
```

Comment:
Have you double checked that it would not display an unexpected scrollbar (since Macos per default hide scrollbars and if not expected)?